plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -294,6 +297,10 @@ public Instance startService(String namespace,
 
             docker.startContainer(StartContainerParams.create(container));
 
+            if (System.getenv("CHE_USER_ID") != null) {
+                changeWorkingDirOwner(container, machineLogger);
```

Comment:
When che starts a workspace, it expects that there is the "user" user that has the /home/user directory as the home directory with a specific content.
You can check that by the following:
```
docker run -it codenvy/ubuntu_jdk8 id
docker run -it codenvy/ubuntu_jdk8 ls -la /home/user
```

The changeWorkingDirOwner method sets the same environment for a user defined with CHE_USER